.travis.yml

```diff
@@ -11,9 +11,23 @@ matrix:
     - python: pypy3
     - language: generic
       env: PYPY_NIGHTLY_BRANCH=py3.6
+    # Qemu tests are also slow
+    # The unique thing this provides is testing on the given distro's
+    # kernel, which is important when we use new kernel features. This
+    # is also good for testing the latest openssl etc., and getting
+    # early warning of any issues that might happen in the next Ubuntu
+    # LTS.
+    - language: generic
+      # We use bionic for the host, b/c rumor says that Travis's
+      # 'bionic' systems have nested KVM enabled.
+      dist: bionic
+      env:
+        - "JOB_NAME=Ubuntu 19.10, full VM"
```

Comment:
codecov does not like spaces here, and only shows "Ubuntu" - see https://codecov.io/gh/python-trio/trio/commit/7cabd162b3fe04bc1465cecca0a7b07804567210/build

Reply:
However it can't be the exact issue as we have the spaces for the Azure Pipeliens jobs.

Reply:
Hmm, it looks like something is going wrong at the Travis level... at the very top of the file, it shows the initial environment for the job as created by travis, and [that says](https://travis-ci.org/python-trio/trio/jobs/608056847#L216):

```
JOB_NAME=Ubuntu
```

Maybe I need to add more quotes to the YAML or something?

Reply:
Yeah, adding more quotes seems to have fixed it.

Reply:
Nice!